components/access_history_modal/index.ts

```diff
@@ -2,20 +2,22 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
+import {bindActionCreators, Dispatch} from 'redux';
 import {getUserAudits} from 'mattermost-redux/actions/users';
 import {getCurrentUserId, getUserAudits as getCurrentUserAudits} from 'mattermost-redux/selectors/entities/users';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {GenericAction} from 'mattermost-redux/types/actions';
 
-import AccessHistoryModal from './access_history_modal.jsx';
+import AccessHistoryModal from './access_history_modal';
 
-function mapStateToProps(state) {
+function mapStateToProps(state: GlobalState) {
```

Comment:
@devinbinnie Looks like `GlobalState` is causing crash on master.

Reply:
@saturninoabril Working perfectly fine for me, what's the error you're getting?
TS symbols shouldn't even factor into the produced JavaScript, they're stripped out by `babel`

Reply:
Hmm, I got it earlier consistently but not now.  It must be me only.  Sorry for the noise and thanks for looking.